source/_components/sensor.countdown_timer.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: "Countdown Timer"
+description: "Countdown Time for Home Assistant"
+date: 2016-12-20 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+
+**Description:**
+Countdown Timer for [Home Assistant](https://github.com/home-assistant/home-assistant)
+
+Combines one or more binary_sensors (typically PIR sensors) 
+and a switch to provide a switch that automatically
+turns off after a specified delay time. 
+
+Creates a sensor that displays the time remaining before the
+switch is turned off. 
+
+If the sensors detect activity while the switch is on then the timer is
+restarted.
+
+The can be accomplished otherwise in Home Assistant. However this component reduces
+the user configuration from dozens of lines of automation and scripting
+to just a few lines in the configuration.yaml file.
+
+````yaml
+# Example configuration.yaml entry
+sensor countdown_timer:
+  platform: countdown_timer
+  timers:
+    lr_light:
+      sensors: binary_sensor.lr_pir1
+      switch:  switch.lr_light_switch
+````
+
+- **timers** (*Required*): The array that contains all timers.
+  - **identifier ** (*Required*): Name of the timer. Multiple entries are possible.  
+    - **sensors** (*Required*): One or more binary sensor (comma separated).
+    - **switch** (*Required*): Light (or other) switch.
+    - **delay** (*Optional*): delay time. default: 20 seconds.
+    - **restart** (*Optional*): True/False, restart timer when HA is restarted. Default: False
+
```

Comment:
Part below this isn't needed in this documentation. In general documentation pages for components are minimized.